GeneralsMD/Code/GameEngine/Source/GameClient/GUI/GUICallbacks/Menus/ScoreScreen.cpp

```diff
@@ -541,23 +541,35 @@ WindowMsgHandledType ScoreScreenSystem( GameWindow *window, UnsignedInt msg,
 			{
 				TheShell->pop();
 				TheCampaignManager->setCampaign(AsciiString::TheEmptyString);
+				if (!TheGlobalData->m_simulateReplays.empty())
+				{
+					TheWritableGlobalData->m_showReplayContinueButton = FALSE; // Tell GameEngine to quit.
```

Comment:
I do not understand this. Isn't `TheGameEngine->setQuitting` telling game engine to quit?